src/pencilkit.cs

```diff
@@ -0,0 +1,243 @@
+
+		// This exists in the base class
+		// [Export ("delegate", ArgumentSemantic.Weak), NullAllowed]
+		// NSObject WeakDelegate { get; set; }
+
+		[Wrap ("WeakDelegate"), NullAllowed, New]
+		IPKCanvasViewDelegate Delegate { get; set; }
+
+		[Export ("drawing", ArgumentSemantic.Copy)]
+		PKDrawing Drawing { get; set; }
+
+		[Export ("tool", ArgumentSemantic.Copy)]
+		PKTool Tool { get; set; }
+
+		[Export ("rulerActive")]
+		bool RulerActive { [Bind ("isRulerActive")] get; set; }
+
+		[Export ("drawingGestureRecognizer")]
+		UIGestureRecognizer DrawingGestureRecognizer { get; }
+
+		[Export ("allowsFingerDrawing")]
+		bool AllowsFingerDrawing { get; set; }
+	}
+
+	[iOS (13, 0), Mac (10, 15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface PKDrawing : NSCopying, NSSecureCoding {
+
+		[Field ("PKAppleDrawingTypeIdentifier")]
+		NSString AppleDrawingTypeIdentifier { get; }
+
+		[DesignatedInitializer]
+		[Export("init")]
+		IntPtr Constructor();
+
+		[DesignatedInitializer]
+		[Export ("initWithData:error:")]
+		IntPtr Constructor (NSData data, [NullAllowed] out NSError error);
+
+		[Export ("dataRepresentation")]
+		NSData DataRepresentation { get; }
+
+		[Export ("bounds")]
+		CGRect Bounds { get; }
+
+		[Export ("imageFromRect:scale:")]
+		UIImage GetImage (CGRect rect, nfloat scale);
+
+		[Export ("drawingByApplyingTransform:")]
+		PKDrawing GetDrawing (CGAffineTransform transform);
+
+		[Export ("drawingByAppendingDrawing:")]
+		PKDrawing GetDrawing (PKDrawing drawing);
+	}
+
+	[iOS (13, 0), NoMac]
+	[BaseType (typeof (PKTool))]
+	[DisableDefaultCtor]
+	interface PKEraserTool {
+
+		[Export ("eraserType")]
+		PKEraserType EraserType { get; }
+
+		[DesignatedInitializer]
+		[Export ("initWithEraserType:")]
+		IntPtr Constructor (PKEraserType eraserType);
+	}
+
+	[iOS (13, 0), NoMac]
+	[BaseType (typeof (PKTool))]
+	[DisableDefaultCtor]
+	interface PKInkingTool {
+
+		[DesignatedInitializer]
+		[Export ("initWithInkType:color:width:")]
+		IntPtr Constructor ([BindAs (typeof (PKInkType))] NSString type, UIColor color, nfloat width);
+
+		[Export ("initWithInkType:color:")]
+		IntPtr Constructor ([BindAs (typeof (PKInkType))] NSString type, UIColor color);
+
+		[Static]
+		[Export ("defaultWidthForInkType:")]
+		nfloat GetDefaultWidth ([BindAs (typeof (PKInkType))] NSString inkType);
+
+		[Static]
+		[Export ("minimumWidthForInkType:")]
+		nfloat GetMinimumWidth ([BindAs (typeof (PKInkType))] NSString inkType);
+
+		[Static]
+		[Export ("maximumWidthForInkType:")]
+		nfloat GetMaximumWidth ([BindAs (typeof (PKInkType))] NSString inkType);
+
+		[Export ("inkType")]
+		[BindAs (typeof (PKInkType))]
+		NSString InkType { get; }
+
+		[Export ("color")]
+		UIColor Color { get; }
+
+		[Export ("width")]
+		nfloat Width { get; }
+	}
+
+	[iOS (13, 0), NoMac]
+	[BaseType (typeof (PKTool))]
+	[DisableDefaultCtor]
+	interface PKLassoTool {
+
+		[DesignatedInitializer]
+		[Export("init")]
```

Comment:
```suggestion
		[Export ("init")]
```

Reply:
same, use the simpler `[DesignatedDefaultCtor]`